src/main/java/uk/gov/hmcts/reform/et/syaapi/service/NotificationService.java

```diff
@@ -390,114 +377,87 @@ public void sendAcknowledgementEmailToTribunal(CaseData caseData,
         );
-
     }
 
     /**
      *  Send acknowledgment email to the claimant when they are responding to
      *  an application (type A/B) made by the Respondent.
      *
-     * @param caseData        existing case details
-     * @param claimant        claimant's full name
-     * @param caseNumber      ethos case reference
-     * @param respondentNames concatenated respondent names
-     * @param hearingDate     date of the nearest hearing
-     * @param caseId          16 digit case id
+     * @param details core details of the email
      * @param applicationType type of application
      * @param copyToOtherParty  whether to notify other party
      * @param isRespondingToRequestOrOrder indicates whether the reply is to a tribunal order or not
      */
-    public void sendResponseEmailToClaimant(
-        CaseData caseData,
-        String claimant,
-        String caseNumber,
-        String respondentNames,
-        String hearingDate,
-        String caseId,
-        String applicationType,
-        String copyToOtherParty,
-        boolean isRespondingToRequestOrOrder
-    ) {
+    void sendResponseEmailToClaimant(CoreEmailDetails details, String applicationType, String copyToOtherParty,
+                                     boolean isRespondingToRequestOrOrder) {
         if (TYPE_C.equals(applicationType)) {
             log.info("Type C application -  Claimant is only notified of "
                          + "Type A/B application responses, email not being sent");
             return;
         }
-        if (isBlank(caseData.getClaimantType().getClaimantEmailAddress())) {
+        String claimantEmailAddress = details.caseData.getClaimantType().getClaimantEmailAddress();
+        if (isBlank(claimantEmailAddress)) {
             log.info("No claimant email found - Application response acknowledgment not being sent");
             return;
         }
         Map<String, Object> claimantParameters = new ConcurrentHashMap<>();
 
-        String subjectLine = String.format("%s %s", caseNumber, applicationType);
+        String subjectLine = String.format(TWO_STRINGS_PATTERN, details.caseNumber, applicationType);
 
         addCommonParameters(
             claimantParameters,
-            claimant,
-            respondentNames,
-            caseId,
-            caseNumber,
+            details.claimant,
+            details.respondentNames,
+            details.caseId,
+            details.caseNumber,
             subjectLine,
             applicationType
         );
         claimantParameters.put(
             SEND_EMAIL_PARAMS_HEARING_DATE_KEY,
-            hearingDate
+            details.hearingDate
         );
         claimantParameters.put(
             SEND_EMAIL_PARAMS_CITIZEN_PORTAL_LINK_KEY,
-            String.format(CONCAT2STRINGS, notificationsProperties.getCitizenPortalLink(), caseId)
+            String.format(CONCAT2STRINGS, notificationsProperties.getCitizenPortalLink(), details.caseId)
```

Comment:
same here, I see above its "%s%s". We're basically just doing a string join here right?

Reply:
Haha yeah, fixing